### DISCUSSION TITLE
is it not needed to close the browser

### DISCUSSION THREAD

**Discussion by iBiocide** [moonD4rk/HackBrowserData#445] [Q&A]
hi, since it is grabbing the cookies and if the browser is open those files are locked, is it running to issues for getting the cookies?
is it not needed to close the browser before running the app ?
would it get all the cookies if the browser is running already?

**Comment by moonD4rk** [comment]
Yes, in the version of Chrome for Windows, the browser has added a lock to cookies, so another method needs to be considered for decryption. I will update in [v0.5.0](https://github.com/moonD4rk/HackBrowserData/issues/437) to support this feature.
